staging/src/github.com/kubeedge/beehive/pkg/core/context/context_channel.go

```diff
@@ -80,7 +80,7 @@ func (ctx *ChannelContext) Receive(module string) (model.Message, error) {
 	}
 
 	klog.Warningf("Failed to get channel for module:%s when receive message", module)
-	return model.Message{}, fmt.Errorf("failed to get channel for module(%s)", module)
+	return model.Message{}, fmt.Errorf("Failed to get channel for module(%s)", module)
```

Comment:
The first letter of the `log` output needs to be capitalized, but the `error` not recommended

Reply:
+1 :)